angler/config/system-proprietary-blobs-api24.txt

```diff
@@ -26,12 +31,17 @@ system/lib/libmmjpeg_interface.so
 system/lib/libqomx_core.so
 system/lib/libtinyxml.so
 system/lib/soundfx/libfmas.so
+system/lib64/hw/activity_recognition.angler.so
 system/lib64/hw/audio.primary.msm8994.so
+system/lib64/hw/context_hub.default.so
```

Comment:
The AOSP build indeed differs from factory image. Do you have any idea why? Have you spotted any issues when using the AOSP version?